gitlab-pipeline/stage/test.yml

```diff
@@ -201,12 +206,17 @@ test:backend-integration:azblob:enterprise:
     - apk --update add curl jq sysstat hdparm
 
     # Start dockerd in the background
-    - /usr/local/bin/dockerd &
+    # The entrypoint script additionally uses docker-init as workarround of known
+    # containerd issue - https://github.com/docker-library/docker/issues/318
+    - export TINI_SUBREAPER=true
+    - /bin/sh -c "/usr/local/bin/dockerd-entrypoint.sh &"
+
     # Wait for dockerd to start
     - |-
       MAX_WAIT=30
       while [ $MAX_WAIT -gt 0 ]; do
-        docker version && break
+        [ -n "${DEBUG_CI}" ] && ( echo "[$(date +%F_%T)] MAX_WAIT=${MAX_WAIT}"; ps afx )
```

Comment:
We should then document this variable in [here](https://github.com/mendersoftware/mender-qa/blob/3e2913a6f1fce4823d46b24443151e6b7e6d163c/.gitlab-ci.yml#L1), don't you think?

We could also drop this completely for not having one variable for one particular thing that we don't plan to debug again. I don't have an strong opinion.

Reply:
Yes, probably introducing a new variable used in one place is not the best idea, if we don't have any other places where now we have excessive debug logging.